pilot/pkg/dns/monitoring.go

```diff
@@ -30,14 +30,14 @@ var (
 	)
 
 	failures = monitoring.NewSum(
-		"dns_total_upstream_failures",
+		"dns_upstream_failures_total",
 		"Total number of DNS requests forwarded to upstream.",
 	)
 
 	requestDuration = monitoring.NewDistribution(
-		"dns_upstream_request_time_ms",
-		"Total time in milliseconds Istio takes to get DNS response from upstream.",
-		[]float64{1, 3, 5, 10, 25, 50},
+		"dns_upstream_request_time",
+		"Total time in seconds Istio takes to get DNS response from upstream.",
+		[]float64{.01, .1, 0.5, 1, 3, 5},
```

Comment:
The buckets seem pretty skewed to extreme latency, while before we had max 50ms. I think we could probably have something like:
`[]float64{.005, .001, 0.01, 0.1, 1, 5},`

WDYT?

Reply:
Makes sense will change